ALLHUBS-Jan-2024-Liftoff/code-stars
# 9 review view

In this branch, I:
- Created the DisplayReview component, which takes in a review ID as props and shows the corresponding review
- Styled the review with bootstrap
- Made a DisplayRating component, which takes in a rating as props and returns stars (this was supposed to be part of the rating branch, oops!)
- Renamed the reviewForm folder to Review to keep the front end files organized
- Put two test reviews on the campground page

In order to test it, please:
1. Start all servers
2. Make sure there's at least 2 reviews in your database, with ids matching the test reviews on lines 15 and 16 of campground.jsx. If not, create reviews and/or change the ids in VSCode.
3. Go to the campground page and make sure reviews render, including feedback and rating stars

They won't display usernames yet. I wanted to get this code out of the door so we have something to start with. I'll be tweaking it this week as more features roll out.
